### DISCUSSION TITLE
SwiftUI LottieView disappears when previewing preview with contextMenu

### DISCUSSION THREAD

**Discussion by chuymaster** [airbnb/lottie-ios#2355] [General]
When using [contextMenu with preview](https://developer.apple.com/documentation/swiftui/view/contextmenu(menuitems:preview:)), if I long-press to open the context menu, Lottie view disappears, causing glitch in view transition.

Refer to the video:

https://github.com/airbnb/lottie-ios/assets/8290104/69aef245-2f0b-4823-8ba3-bea47081d610

Reproducing code, tested on iOS 17.2 simulator. (Confirmed same behavior on iPhone 15 iOS 17.4) 

```swift
import SwiftUI
import Lottie

struct ContentView: View {
    var body: some View {
        VStack {
            Text("Normal Context Menu")
            LottieView(animation: .named("animation"))
                .looping()
                .contextMenu(menuItems: {
                    Text("Menu Item 1")
                    Text("Menu Item 2")
                    Text("Menu Item 3")
                })
            Text("Context Menu with Preview")
            LottieView(animation: .named("animation"))
                .looping()
                .contextMenu(menuItems: {
                    Text("Menu Item 1")
                    Text("Menu Item 2")
                    Text("Menu Item 3")
                }) {
                    Text("Context Menu Preview")
                }
        }
        .padding()
        .background(Color.gray.opacity(0.1))
    }
}
```

## Which Version of Lottie are you using?

Lottie 4.4.1

## Expected Behavior 

LottieView keeps running animation during long-press and after the preview is closed, as in the context menu that has no preview.

<img width=300 src="https://github.com/airbnb/lottie-ios/assets/8290104/dfceed96-fdae-48aa-bbc3-02d72c5eb873)" />

## Actual Behavior

LottieView disappears during long-press and after the preview is closed for a short period.

<img width=300 src="https://github.com/airbnb/lottie-ios/assets/8290104/2c911e23-4668-4c0e-960d-449387015f51" />

## Animation JSON

https://lottiefiles.com/animations/sample-ac0sBj9eCw

**Comment by pablozin** [comment]
Have you tried to put the LottieView inside a ZStack?
